test/setupServer.ts

```diff
@@ -0,0 +1,23 @@
+/* istanbul ignore file */
+import http from 'http';
+import { parse } from 'url';
+import express from 'express';
+import type { NextApiHandler } from 'next';
+import { apiResolver } from 'next/dist/next-server/server/api-utils';
+
+export function setupServer(handler: NextApiHandler): http.Server | express.Express {
+  if (process.env.TEST_SERVER === 'nextjs') {
+    console.log('[i] Using the Next.js server.');
```

Comment:
Not sure if we should add this. If we do, I feel like we also call out express servers to create a concrete seperation between the two. But I think removing it would be just fine.

Reply:
Yeah, I agree. Removed it.